tools/ete/ete_lineage_generator.xml

```diff
@@ -20,6 +20,7 @@ python '$__tool_directory__/ete_lineage_generator.py'
     $compresscond.compress
```

Comment:
Need to increase the tool version (e.g. add `+galaxy1`) on the first line.

Reply:
I always forget this...